### PR TITLE
FWF507: fix sort outsideclick

### PR DESCRIPTION
**Pull request by kurianbenoy-aot**
Issue Description: Unable to remove the list on clicking on filter option ""Created"

UI screen:

image

### CODE REVIEW

File: camunda-formio-tasklist-vue/src/components/layout/Header.vue
Comment (ann-aot) [inline camunda-formio-tasklist-vue/src/components/layout/Header.vue:27]: pls correct the spacing

**Review by ann-aot — APPROVED** [review]

**Comment by ann-aot** [comment]
there are some conflicts- pls resolve the conflicts

**Comment by kurianbenoy-aot** [comment]
@ann-aot please don't merge this PR for the time being

File: camunda-formio-tasklist-vue/src/components/layout/Header.vue
Comment (kurianbenoy-aot) [reply camunda-formio-tasklist-vue/src/components/layout/Header.vue]: fixed the spacing issue

**Comment by kurianbenoy-aot** [comment]
I have resolved the conflict and fixed the issues after merging. Now the outside click is working fine

image

**Comment by kurianbenoy-aot** [comment]
Merging #7

**Pull request merged**
